superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx

```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
             name: 'normalize_across',
             config: {
               type: 'SelectControl',
-              label: t('Normalize Across'),
+              label: t('Color based on'),
```

Comment:
I think I might be OK with "Normalize Across" now that we know it's actually doing normalization